serverless.yml

```diff
@@ -63,6 +63,13 @@ functions:
           method: POST
           path: /create-customer/{ID}
           cors: true  
+  createStrictCustomer:
+    handler: src/handlers/createStrictCustomer.handler
+    events:
+      - http:
+          method: POST
+          path: /create-strict-customer/{ID}
```

Comment:
I want you to think about what a good path should look like. Check out some articles on REST API conventions

You'll find that you shouldn't have any verbs in there. The method should tell you enough